docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,544 @@
+# Create a channel without a system channel
+
+Fabric v2.3 introduces the ability to create channels between network members without requiring an ordering service system channel. This feature greatly simplifies the channel creation process and provides a mechanism for Raft consenters to join a channel with zero downtime. Use this tutorial to learn how to create new channels without a system channel by using the [configtxgen](../commands/configtxgen.html) tool along with the [osnadmin CLI](../commands/osnadminchannel.html).
+
+**How this process differs from the legacy process:**
+
+ * **System channel no longer required**: Previously a channel managed by the ordering service (called the "system channel") was defined using the `configtxgen` tool to create a genesis block. This channel was exclusively managed by the ordering service and was the template on which all application channels were based. Besides the creation of the system representing an extra step (as compared to the new process), this system channel created an extra layer of administration that, for some use cases, provided no tangible benefit.
+ * **Consortium no longer required**: You no longer need to define the set of organizations, known as the "consortium", who are permitted to create channels on a particular ordering service. With this new process, **all channels are application channels**, so the concept of a list of organizations who can create channels no longer applies. Any set of organizations can get together and create a channel using a defined set of ordering nodes (which become the ordering service of that channel). Additionally, if the MSP of the peer organization, as listed in the system channel, needs to be changed, the peer organization no longer needs to coordinate with an orderer organization to change the MSP.
```

Comment:
The `all channels are application channels` bit isn't technically accurate, but for the sake of this document it's probably what we want to include/recommend. You can still create a system channel genesis block and use the API to create the system channel (or onboard a new orderer to an existing system channel).

Reply:
This is an important point - thank you for mentioning it because I had not really considered it. My thinking though is that since this is a tutorial about creating a channel without a system channel, it doesn't really fit in this topic per se and could be confusing. I think we should def include this as appropriate when we update all the other topics that talk about the system channel though.